solutions/instances/README.md

```diff
@@ -0,0 +1,14 @@
+# IBM Observability Instances Deployable Architecture
+
+This architecture creates the observability instances on IBM Cloud and supports provisioning the following resources:
+
+- A resource group, if one is not passed in.
+- A log analysis instance.
+- A cloud monitoring instance.
+- Creates a cloud object storage(COS) instance or taking in an existing one.
+- Creates a KMS encrypted COS bucket that is required to store archived logs or using the existing one.
+- Creates a KMS encrypted COS bucket for setting up Activity Tracker event routing or using the existing one.
```

Comment:
You need to add a bullet point mentioning that the DA supports creating KMS root keys in an existing KMS instance or using existing keys if creating new buckets